metrics/metrics-core/pom.xml

```diff
@@ -29,18 +29,27 @@
   <artifactId>metrics-core</artifactId>
 
   <dependencies>
+    <dependency>
+      <groupId>com.netflix.servo</groupId>
+      <artifactId>servo-core</artifactId>
+    </dependency>
     <dependency>
       <groupId>io.servicecomb</groupId>
-      <artifactId>java-chassis-core</artifactId>
+      <artifactId>metrics-common</artifactId>
     </dependency>
     <dependency>
       <groupId>io.servicecomb</groupId>
-      <artifactId>foundation-metrics</artifactId>
+      <artifactId>provider-rest-common</artifactId>
```

Comment:
do not use rest-common + spring web
please use provider springmvc

Reply:
Done